crux-mir/translate_libs.sh

```diff
@@ -84,8 +93,14 @@ translate lib/hashbrown/src/lib.rs --edition=2021 --crate-name hashbrown --cfg '
 echo 'Building addr2line...'
 translate lib/addr2line/src/lib.rs  --crate-name addr2line --cfg 'feature="alloc"' --cfg 'feature="compiler_builtins"' --cfg 'feature="core"' --cfg 'feature="rustc-dep-of-std"' --extern "compiler_builtins=${RLIBS}/libcompiler_builtins.rlib" --extern "gimli=${RLIBS}/libgimli.rlib" --extern "alloc=${RLIBS}/liballoc.rlib" --extern "core=${RLIBS}/libcore.rlib"
 
+if [ "$TARGET_ARCH" = "wasm32" ] && [ "$TARGET_OS" != "emscripten" ]; then
+  USE_DLMALLOC=yes
+  echo 'Building dlmalloc...'
+  translate lib/dlmalloc/src/lib.rs --crate-name dlmalloc --cfg 'feature="rustc-dep-of-std"' --extern "compiler_builtins=${RLIBS}/libcompiler_builtins.rlib" --extern "core=${RLIBS}/libcore.rlib"
+fi
```

Comment:
Worth leaving a comment here saying what `dlmalloc` is and why it is only needed for `wasm32`. (Also, how did you figure out that this was needed?)